.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+< A brief description of the pull request >
+
+---
+
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | yes/no
+| Deprecations? | yes/no
+| Tests pass?   | yes/no
+
+---
+
+## Before
+
+< The current status >
+
+## After
+
+< What will be changed >
+
+## Screenshots
```

Comment:
I think screenshots should be included as part of the Before/After sections, otherwise you would have to scroll up/down to read the changes and see the screenshots